### PR TITLE
CORCI-668 build: Add Ubuntu packaging

### PR DESCRIPTION
**Pull request by JohnMalmberg**
Add the debian files needed to create Ubuntu 18 packages.

Signed-off-by: John.E.Malmberg <john.e.malmberg@intel.com>

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Description: Data Plane Development Kit (basic development files)
+ Fork of DPDK from the SPDK project.
+ DPDK is a set of libraries for fast packet processing. Applications run
+ in user-space and communicate directly with dedicated network interfaces.
+ .
+ This package contains the basic headers and library files required to
+ build external applications which will also require at least SSE3 support
+ when running.
+
+Package: dpdk1
```
Comment (brianjmurrell) [inline debian/control:62]: Ditto.

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+         ${shlibs:Depends},
+         ${misc:Depends}
+Description: Data Plane Development Kit (basic development files)
+ Fork of DPDK from the SPDK project.
+ DPDK is a set of libraries for fast packet processing. Applications run
+ in user-space and communicate directly with dedicated network interfaces.
+ .
+ This package contains the runtime environment to run DPDK applications.
+
+Package: dpdk-dev
```
Comment (brianjmurrell) [inline debian/control:46]: Is this the name Ubuntu-native packages are using?

Comment (JohnMalmberg) [reply debian/control:46]: No.  This DPDK fork from SPDK is totally incompatible with any libdpdk packages that Ubuntu is providing.

**Comment by JohnMalmberg** [comment]
Obsolete

**Pull request closed without merging**